content/blog/2025/12/10/2025-12-10-jenkins-bug-bounty-program-yeswehack-european-commission.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Announcing the new Jenkins Bug Bounty Program"
+tags:
+- jenkins
+- community
+- security
+- yeswehack
+- europeancommission
+authors:
+- wadeck
+opengraph:
+  image: /images/post-images/2025/12/yeswehack-jenkins.jpg
+discourse: true
+---
+It is with great pleasure that we announce the new Jenkins Bug Bounty Program!
+The European Commission (link:https://interoperable-europe.ec.europa.eu/collection/ec-ospo[EC OSPO]) has partnered with link:https://www.yeswehack.com/[YesWeHack] to launch bug bounty programs for several open source projects.
+The Jenkins project was selected as a valuable asset for public administration across the European Union.
+
+The program will run for one year, rewarding security researchers who find and responsibly disclose security vulnerabilities in Jenkins.
+
+== Why a Bug Bounty Program?
+
+Bug bounty programs complement our existing security practices by engaging the security research community in identifying and responsibly disclosing vulnerabilities.
+
+This additional layer of scrutiny, combined with financial incentives, helps ensure Jenkins remains a secure and reliable automation server for organizations worldwide.
+
+== Program Details
+
+* *Initial scope*: Jenkins Core and its main components, and four plugins related to security
+* *Reward*: Up to €5,000 for valid critical findings!
+* *Platform*: link:https://yeswehack.com/programs/jenkins-bug-bounty-program[Jenkins Bug Bounty Program on YesWeHack]
+* *Funding*: European Commission
+
+Depending on the level of interest and quality of reports, we will consider expanding the scope to other components of the Jenkins ecosystem.
+
+For security researchers, it's an additional reason to invest time on Jenkins. In addition to the white-box testing enabled by the open-source nature of the project, you can even get financial rewards for your efforts!
+
+== How Does It Work?
+- The in-scope reports are sent to the YesWeHack platform.
+- The initial triage is done by YesWeHack team.
```

Comment:
out of curiosity does their triage including reproducing issues or just saying this doesn't look like junk I'll pass it on?